intermine/registry.py

```diff
@@ -0,0 +1,50 @@
+import requests
+import yaml
+from intermine.webservice import Service
+
+def getInfo(mine):
+    link="http://registry.intermine.org/service/instances/" + mine
+    r=requests.get(link)
+
+    dict=yaml.load(r.text)
+    print("Description: " + dict["instance"]["description"])
+    print("URL: " + dict["instance"]["url"])
+    print("API Version: " + dict["instance"]["api_version"])
+    print("Release Version: " + dict["instance"]["release_version"])
+    print("InterMine Version: " + dict["instance"]["intermine_version"])
+    print("Organisms: "),
+    for organism in dict["instance"]["organisms"]:
+        print(organism),
+    print("Neighbours: "),
+    for neighbour in dict["instance"]["neighbours"]:
+        print(neighbour),
+
+def getData(x):
+    link = "http://www." + x + ".org/" + x + "/service"
```

Comment:
This works great for some intermines, but not all InterMines follow this url schema - e.g. not http, running on subdomains (like http://maizemine.rnet.missouri.edu:8080/maizemine ), or even something like bovinemine, on http://bovinegenome.org/bovinemine